src/parsers/manifest/dash/common/parse_adaptation_sets.ts

```diff
@@ -424,6 +424,9 @@ export default function parseAdaptationSets(
       if (isSignInterpreted === true) {
         parsedAdaptationSet.isSignInterpreted = true;
       }
+      if (label) {
```

Comment:
To limit unforeseen false positives/negatives (for example here the empty string would not set that value, though maybe this is wanted), we generally prefer to enforce that only boolean conditions are checked in an `if`.
So for example, if what is really wanted here is just to check that `label` is defined on parsedAdaptationSet, we would prefer something like `if (label !== undefined)`.